tools/GraphClust/GSPAN/fasta2shrep_gspan.xml

```diff
@@ -23,15 +23,28 @@
 		<param name="wins" type="text" value="40,150" size="20" label=" A list of window sizes to use" help="comma separated integers"/>
 		<param name="shift" type="integer" value="30" size="7" label="The shift of the window, relative to the window size given inpercent." help="by default 30"/>
 		<param name="u" truevalue="-u" falsevalue="" checked="True"  type="boolean" label="Ignore unstable structures (RNAshapes)" help="-u"/>
-		<param name="seq_graph_t" truevalue="--seq-graph-t" falsevalue="" checked="True"  type="boolean" label="Add for each 't #' a graph which contains no structure" help="--seq-graph-t"/>
+		<param name="seq_graph_t" truevalue="-seq-graph-t" falsevalue="" checked="True"  type="boolean" label="Add for each 't #' a graph which contains no structure" help="--seq-graph-t"/>
```

Comment:
actually it's with one, but i noticed that only today, so before it was working also with 2.